app/src/main/java/swati4star/createpdf/fragment/MergeFilesFragment.java

```diff
@@ -111,14 +110,15 @@ public void onInput(@NonNull MaterialDialog dialog, CharSequence input) {
     private void mergePdfFiles() {
         try {
 
-            String filepath1 = filepathtv1.getText().toString();
-            String filepath2 = filepathtv2.getText().toString();
+            String filepath1 = filepathtv1;
```

Comment:
There is no need of these variables now ( `filepath1` , `filepath2` )
You can remove these & directly use `filepathtv1` & `filepathtv2`